src/http/station.ts

```diff
@@ -1324,6 +1324,29 @@ export class Station extends TypedEmitter<StationEvents> {
         }
     }
 
+    public async setMotionZone(device: Device, value: string): Promise<void> {
+        const propertyData: PropertyData = {
+            name: PropertyName.DeviceMotionZone,
+            value: value
+        };
+        if (device.getStationSerial() !== this.getSerial()) {
+            throw new WrongStationError(`Device ${device.getSerial()} is not managed by this station ${this.getSerial()}`);
+        }
+        if (!device.hasProperty(propertyData.name)) {
+            throw new NotSupportedError(`This functionality is not implemented or supported by ${device.getSerial()}`);
+        }
+        this.log.debug(`Sending motion zone command to station ${this.getSerial()} for device ${device.getSerial()} with value: ${value}`);
+
+        await this.p2pSession.sendCommandWithStringPayload({
+            commandType: CommandType.CMD_DOORBELL_SET_PAYLOAD,
+            value: JSON.stringify({
+                "commandType": CommandType.CMD_INDOOR_DET_SET_ACTIVE_ZONE,
+                "data": JSON.parse(value)
```

Comment:
The JSON requires to be parsed here as it will otherwise be interpreted as a string and `value` becomes quoted `"`